apps/api/src/services/__tests__/perps-data-processor-orchestration.test.ts

```diff
@@ -1,1003 +0,0 @@
-import { beforeEach, describe, expect, it, vi } from "vitest";
```

Comment:
what happened to this test and the other tests in this directory?  They seem to just be deleted?

Reply:
oh I think they were moved but git got confused.  If you use `git mv` instead of regular `mv`, then git can track files across moves much better and it makes reviewing changes like this a lot easier